iree/compiler/Codegen/Passes.td

```diff
@@ -172,6 +172,18 @@ def MemrefCopyToLinalgPass :
   let constructor =
       "mlir::iree_compiler::createMemrefCopyToLinalgPass()";
 }
+
+def WorkGroupSwizzle :
```

Comment:
Not sure why there are two passes. Maybe make one a utility function and another a pass?

Reply:
This is only one pass right?

Reply:
Ah sorry my bad....